### PR TITLE
TLF-321-changes-notify

### PR DESCRIPTION
**Pull request by RobertMcCann**
Relates [TLF-321](https://collaboration.homeoffice.gov.uk/jira/browse/TLF-321)
## What? 
Add keys to capture tenant/rental address for business email
## Check list

- [ ] I have reviewed my own pull request
- [ ] I have written tests (if relevant)

### CODE REVIEW

File: apps/lcs/behaviours/send-email-notification.js [hunk 1]
```diff
@@ -39,10 +39,25 @@ const getPersonalisation = (recipientType, req) => {
     landlord_company: req.sessionModel.get('landlord-or-agent-company'),
     landlord_email: req.sessionModel.get('landlord-or-agent-email'),
     landlord_tel: req.sessionModel.get('landlord-or-agent-tel'),
-    rental_postcode: req.sessionModel.get('valuesEnriched').tenantType === 'Prospective tenant' ?
+    prospective_rental_postcode: req.sessionModel.get('valuesEnriched').tenantType === 'Prospective tenant' ?
```
Comment (dk4g) [inline apps/lcs/behaviours/send-email-notification.js]: use boolean variable`isCurrentTenant` instead

File: apps/lcs/behaviours/send-email-notification.js [hunk 2]
```diff
@@ -39,10 +39,25 @@ const getPersonalisation = (recipientType, req) => {
...
+    rental_address_line_1: req.sessionModel.get('valuesEnriched').tenantType === 'Tenant' ?
+      req.sessionModel.get('tenant-address-line-1') : '',
+    rental_address_line_2: req.sessionModel.get('valuesEnriched').tenantType === 'Tenant' ?
+      req.sessionModel.get('tenant-address-line-2') : '',
+    rental_town_or_city: req.sessionModel.get('valuesEnriched').tenantType === 'Tenant' ?
+      req.sessionModel.get('tenant-town-or-city') : '',
+    rental_county: req.sessionModel.get('valuesEnriched').tenantType === 'Tenant' ?
+      req.sessionModel.get('tenant-county') : '',
+    rental_postcode: req.sessionModel.get('valuesEnriched').tenantType === 'Tenant' ?
+      req.sessionModel.get('tenant-postcode') : '',
```
Comment (jamiecarterHO) [inline apps/lcs/behaviours/send-email-notification.js]: you could use `req.sessionModel.get('isCurrentTenant')` as the condition here too.

File: apps/lcs/behaviours/send-email-notification.js [hunk 3]
```diff
@@ -39,10 +39,25 @@ const getPersonalisation = (recipientType, req) => {
     landlord_company: req.sessionModel.get('landlord-or-agent-company'),
     landlord_email: req.sessionModel.get('landlord-or-agent-email'),
     landlord_tel: req.sessionModel.get('landlord-or-agent-tel'),
-    rental_postcode: req.sessionModel.get('valuesEnriched').tenantType === 'Prospective tenant' ?
+    prospective_rental_postcode: req.sessionModel.get('valuesEnriched').tenantType === 'Prospective tenant' ?
       req.sessionModel.get('rental-property-postcode') : '',
     rental_property_address: req.sessionModel.get('valuesEnriched').tenantType === 'Tenant' ?
-      req.sessionModel.get('tenantAddress').join('\n') : ''
+      req.sessionModel.get('tenantAddress').join('\n') : '',
```
Comment (jamiecarterHO) [inline apps/lcs/behaviours/send-email-notification.js]: Should be joined with `(', ')` to match the figma format. Same for the `full_address` param above. Also not certain this copes with optional fields address line 2 and county too well. However, I think that can be dealt with in the PR for TLF-117

**Review by jamiecarterHO — CHANGES_REQUESTED** [review]
Just some of the conditional stuff on tenantType could be a little neater here. The `full_address` will probably not match the designs for format, but seems more sensible to deal with them alongside the other address aggregation code from the higher PR

**Review by jamiecarterHO — APPROVED** [review]
lgtm

**Pull request merged**